app/decorators/reviewing_asset.rb

```diff
@@ -5,6 +5,11 @@ def review!
     save
   end
 
+  def unreview!
```

Comment:
I'm a fan of avoiding "what if" programming, but I'd say if there's a definitely use-case we are planning, it's a good thing to keep.  If it's a "probably", it might be worth keeping.  But also bear in mind that we should have one way of unreviewing if possible.  I'd rather use attribute magic *or* an explicit function in all three cases, not mix and match.